lib/plugin/instance.rb

```diff
@@ -256,10 +256,10 @@ def add_model_callback(klass_name, callback, options = {}, &block)
     end
   end
 
-  # Add a post_custom_fields_whitelister block to the TopicView, respecting if the plugin is enabled
-  def topic_view_post_custom_fields_whitelister(&block)
+  # Add a post_custom_fields_allowlister block to the TopicView, respecting if the plugin is enabled
+  def topic_view_post_custom_fields_allowlister(&block)
```

Comment:
This is a plugin API, I don't think we should rename it without a deprecation notice. Can we keep the old name working (with deprecation warning)? It can just call the new function.